app/src/main/java/com/quran/labs/androidquran/ui/adapter/TranslationsAdapter.java

```diff
@@ -26,12 +30,23 @@
 
   private final UnicastSubject<TranslationRowData> onClickDownloadSubject = UnicastSubject.create();
   private final UnicastSubject<TranslationRowData> onClickRemoveSubject = UnicastSubject.create();
+  private final UnicastSubject<TranslationRowData> onClickRankUpSubject = UnicastSubject.create();
+  private final UnicastSubject<TranslationRowData> onClickRankDownSubject = UnicastSubject.create();
+
+  private final AppCompatActivity activity;
```

Comment:
Why do you need an activity here? It is generally a bad idea to have a reference to an activity. 
There should be a better way of achieving your goal. Maybe replace with an interface?

Reply:
OK, I admit programmer laziness since were already using `Context`.  Happy to make that a cleaner interface instead...